animalcules.js

```diff
@@ -0,0 +1,130 @@
+const $window = $(window)
+let canvas = {};
+let ctx = {};
+let particles = [];
```

Comment:
By the way, do you make a new class like `TheWorld` to save the `particles`, `ctx` and `canvas` so we don't have to need the above global variables, with that we will have a cleaner program structure, thanks.